exercises/practice/sudoku/.docs/instructions.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+Sudoku is the popular number puzzle of fitting the numbers 1-9 into each column, row,
+and 3x3 block using each exactly once. Write a z3 function that takes a 9x9 list of lists
```

Comment:
I don't think that you need to say "Write a Z3 function ..." since this will be under the Z3 track. Maybe just start the sentence with "Write a function ..."